src/styles/blocks/card.scss

```diff
@@ -0,0 +1,74 @@
+.card {
+  display: flex;
+  width: 200px;
+  height: 408px;
+  flex-direction: column;
+  border-radius: 5px;
+  border: 1px solid $border-color;
+  padding: 32px 16px 16px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    padding-top: 40px;
+  }
+
+  &__code {
+    font-size: 10px;
+    color: $secondary-text-color;
+    padding-top: 4px;
```

Comment:
seems like you use `padding-top` to set the indent between your elements. It's not a good approach, let's use `margin-top` or `margin-bottom`. Fix it everywhere